### PR TITLE
Update & fixes for Readme

### PR DESCRIPTION
**Pull request by Hero9909**
Some minor changes to the general appearance of the readme.

Also, I've added a Legacy section to notify about the latest breaking changes.

I'm not sure if the wording fits like that, do you want to adjust it?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,26 +73,20 @@ the format options for strings (beta support for i64 and f64)
...
-- [ ]: special suppport to format HashMap<String, String> for improved speed
-- [ ]: special suppport to format HashMap<String, &str> for improved speed
-- [ ]: special suppport to format HashMap<&str, &str> for improved speed
+- [ ] get strfmt_map out of Beta and create Format.format_map method
+- [ ] handle sign aware zero padding for numeric types
+- [x] format any Display type
+- [x] stabalize `strfmt_map` and add `format_map` to the `Format` trait.
+- [x] add `f64` method to `Formatter` allowing those using `strfmt_map` to format f64s according to the spec
+- [x] add `i64` method to `Formatter` allowing those using `strfmt_map` to format i64s according to the spec
+- [ ] add `format_f64(&self, HashMap<String, f64>` method to `Format` allowing users to easily format a hashmap of i64 values
```
Comment (vitiral) [inline README.md:82]: I think this (and the one below) should be removed and replaced with:

```
- [x] use DisplayStr trait for formatting, permitting proper formatting of integer types.
```

**Review by vitiral — APPROVED** [review]

**Comment by vitiral** [comment]
let me know if you think this is all the changes for this week and I can push a new minor version.

**Comment by Hero9909** [comment]
I think it's good for now, update as soon as you like.

**Comment by vitiral** [comment]
released as `0.2.1`

**Pull request merged**